pkg/utils/utils.go

```diff
@@ -20,12 +20,12 @@ import (
 )
 
 const WORKING_DIR = "/tmp/_working_dir"
-const ALLINONE_ANNOTATION = "io.openshift.clusterlogging.alpha/allinone"
+const SPLIT_ANNOTATION = "io.openshift.clusterlogging.alpha/splitinstallation"
```

Comment:
I still think that if we intend to not support 'split' log flows in the current incarnation we should drop the concept here all together and look to utilize our pipeline concept.  This is our opportunity to clean up 4.0:
*  It is going to be green deployments, 
* we still need to figure out how to adopt existing clusters which we can address split scenarios then,
* ops doesnt utilize this arrangement now
* Split is the 2% use case and not the 98%

Reply:
Conversation with @tkatarki, he agreed we will remove the notion of 'split' deployment in 4.0. This is  the time for us to clean the slate.

Reply:
This is something we have customers today using though and we agreed in our meeting that we would flip the functionality of not having the annotation from being that we split to being it is all-in-one so that if a customer still wanted to split their installation it would be far less painful than if we didn't provide a way to do so.

Also, when we go to adopt an existing split install cluster if we want to move away from a split install we will need to have a way to move ES data into a consolidated cluster.